lib/python/picongpu/pypicongpu/species/operation/densityprofile/gaussian.py

```diff
@@ -0,0 +1,53 @@
+"""
+This file is part of the PIConGPU.
+Copyright 2024 PIConGPU contributors
+Authors: Brian Edward Marre, Afshari Masoud
+License: GPLv3+
+"""
+
+from .densityprofile import DensityProfile
+from .... import util
+from typeguard import typechecked
+
+
+@typechecked
+class Gaussian(DensityProfile):
+    """
+    Directional guassian density profile
+    """
+
+    gasCenterLeft = util.build_typesafe_property(float)
+    """  """
+    gasCenterRight = util.build_typesafe_property(float)
+
+    gasSigmaLeft = util.build_typesafe_property(float)
+    """the distance from gasCenterLeft until the gas density decreases to its 1/e-th part in meter"""
+
+    gasSigmaRight = util.build_typesafe_property(float)
+    """the distance from gasCenterRight until the gas density decreases to its 1/e-th part in meter"""
+
+    gasFactor = util.build_typesafe_property(float)
+
+    gasPower = util.build_typesafe_property(float)
+
+    vacuumCellsY = util.build_typesafe_property(int)
+
```

Comment:
needs a description and we should remove the mention of y to avoid magic directions in our configurations
```suggestion
    vacuumCellsFront = util.build_typesafe_property(int)
    """number of vaccum Cells in front of foil for laser init
```